nifpga/bitfile.py

```diff
@@ -171,6 +181,61 @@ def __str__(self):
                 "\tOffset: %d\n" % self._offset)
 
 
+class Fxp_Register(Register):
+    """
+    A fixed point control or indicator from the front panel of the top level
+    FPGA VI.
+    """
+    def __init__(self, reg_xml, register):
+
+        self._copy_values_from_register(register)
+        datatype = reg_xml.find("Datatype")
+        fxp_xml = datatype.find("FXP")
+        self._signed = True if fxp_xml.find("Signed").text.lower() == 'true' else False
+        self._overflow = True if fxp_xml.find("IncludeOverflowStatus").text.lower() == 'true' else False
+        self._word_length = int(fxp_xml.find("WordLength").text)
+        self._integer_word_length = int(fxp_xml.find("IntegerWordLength").text)
+
+    def _copy_values_from_register(self, register):
+        self._name = register.name
```

Comment:
Aren't _name, _offset, and some of the other protected variables already visible since this is an inherited class?

Reply:
This originally was a bad implementation while it was inherited the class did not call the other initialization. Fixing to be more obvious in commit